src/content/datachannel/datatransfer/js/main.js

```diff
@@ -29,6 +29,9 @@ megsToSend.addEventListener('change', function(e) {
   if (this.value <= 0) {
     sendButton.disabled = true;
     errorMessage.innerHTML = '<p>Please enter a number greater than zero.</p>';
+  } else if (this.value > 1000) {
+        errorMessage.innerHTML = '<p>Progress can be slow when the values are greater than 1000.</p>';
```

Comment:
Indentation should match the above `if` statement. Also this is not really an error, its a warning and should be displayed as such.